README.md

```diff
@@ -1,5 +1,6 @@
-# Liquibase database evolution with with Spring Boot 
-Liquibase can not generate ChangeLog from empty database or JPA entities. It should be run against existing database(dev or test) initially before using Liquibase-Hibernate plugin. If you do not have existing database, create new one with 1 table and Liquibase takes it from there. Steps to setup Liquibase from Scratch
+# Liquibase database evolution with with Spring Boot
+Steps to setup Liquibase from Scratch
```

Comment:
The following is needed 

```
Liquibase can not generate ChangeLog from empty database or JPA entities. It should be run against existing database(dev or test) initially before using Liquibase-Hibernate plugin. If you do not have existing database, create new one with 1 table and Liquibase takes it from there. Steps to setup Liquibase from Scratch
```

Reply:
I merged your sources; both branches should auto merge with no problems